.buildkite/autogenerate_pipeline.py

```diff
@@ -237,6 +242,17 @@ def build(self, input):
         if "{target_platform}" in command:
             assert platform, "Command requires platform, but platform is missing."
             command = command.replace("{target_platform}", platform)
+        # Modify command and tag name for `riscv64` CI
+        if platform == "riscv64":
+            # Wrap command with '' to avoid escaping early by `ENTRYPOINT`
+            command = json.dumps(command)
+            # Overwrite image tag for riscv64
+            self.step_config["plugins"][0][f"docker#{DOCKER_PLUGIN_VERSION}"]["image"] = f"rustvmm/dev:{CONTAINER_VERSION_RISCV}"
+            # Set platform to x86_64 for riscv64
+            platform = "x86_64"
+            # Since we are using qemu-system inside the container, we don't need
+            # hypervisor to be passed here
+            hypervisor = ""
```

Comment:
I'd avoid superfluous comments that describe exactly what the code does. Comments should explain why or clarify obscure things, for example here we should explain that we are doing this because the container will be an x86_64 that will run qemu, etc.

Reply:
Noted, I tried to improve the comment a bit, please have a look when convenient ：）

Reply:
much better now, thanks!